README.md

```diff
@@ -1,77 +1,64 @@
+
+The goal of IPLD is to enable decentralized data-structures
+which in turn will enable more decentralized applications.
+
+Many of the specifications in IPLD are inter-dependent.
+
+# IPLD Layer Model
+
+This layer model is a simplified heirarchy of IPLD concepts and requirements.
 
 ```
-                                     The IPLD Stack
-
-           ┌───────────┐ ┌────────────────────────────────────┐
-           │           │ │                                    │
-           │    MFS    │ │       End-User Applications        │
-           │           │ │                                    │
-           ├───────────┤ ├─────────────────────────┬──────────┤            ┌───────────┐
-           │           │ │    Specialized Data     │          │            │           │
-           │ UnixFS v2 │ │ Structures & Utilities: │          │            │    MFS    │
-           │           │ │   VR, Geo, SQL, etc.    │          │            │           │
-           ├───────────┴─┴─────────────────────────┘          │            ├───────────┤
-           │                                                  │            │           │
-           │    Multi-block Collections: Maps, Sets, etc.     │            │ UnixFS v1 │
-           │                                                  │            │           │
-┌──────────┼──────────────────────────────────────────────────┼────────────┼───────────┤
-│          │                                                  │            │           │
-│          │            dag-json         dag-cbor             │  ipld-git  │           │
-│          │                                                  │            │           │
-│  Codecs  ├──────────────────────────────────────────────────┤  ipld-btc  │   dag-pb  │
-│          │                                                  │            │           │
-│          │                 IPLD Data Model                  │ ipld-zcash │           │
-│          │                                                  │            │           │
-└──────────┼──────────────────────────────────────────────────┴────────────┴───────────┤
-           │                                                                           │
-           │                           Block (CID, Raw Data)                           │
-           │                                                                           │
-           └───────────────────────────────────────────────────────────────────────────┘
+┌──────────────────────────────────────────────────┐
+|                                                  | 
+|                   Schema Layer                   |
+|  (advanced types, multi-block data structures)   |
+|                                                  |
+├──────────────────────────────────────────────────┤
+|                                                  |
+|                 Data Model Layer                 |
+|   (basic types, single-block data structures)    |
```

Comment:
I like this explicit distinction: multi-block, single-block, we should be sure to emphasize that

Reply:
i tweaked the wording a bit to make it even more explicit.